src/CLHEPrandomgenerator.cpp

```diff
@@ -37,6 +37,7 @@
 
 #include "CLHEPrandomgenerator.h"
 #include <CLHEP/Random/Randomize.h>
+#include <CLHEP/Random/Random.h>
```

Comment:
Maybe it is come from calling CLHEP.
In another server these lines were worked.
If I can call CLHEP successfully, it will be work.